python-lib/osisoft_plugin_common.py

```diff
@@ -451,6 +451,17 @@ def fields_selector(data_type):
         return "Links%3BItems.Timestamp%3BItems.Value%3BItems.Type"
 
 
+def get_next_page_url(json):
```

Comment:
Will this work for all kinds of PI requests? If not perhaps it needs a comment or a different name.

Reply:
It is generic yes, not always used but present in all PI responses